README.md

```diff
@@ -62,6 +62,38 @@ $ make clean
 
 will remove build artifacts.
 
+### Frontend
+
+#### Set Up & Running Development Server
+
+1. **Follow all the installation and set up instructions above**
```

Comment:
The existing instructions in the README use bolding this way:

```
 6. **Commit** changes to your branch (never to master)
```

which is inconsistent with the style you have here. I think the style needs to be consistent, otherwise the README will be harder for people to read and less clear. I think there are two options:

1. Change the existing style to match what you have here
2. Change the style you use here to match the existing style

I'm okay with either-- just go with the one that you think makes it easiest for people to read and understand the README.

Reply:
I'll edit my added text to match the conventions followed by the existing style (e.g. the 'Contributing' section).